### ISSUE TITLE
[Duplications] [1168] Error on the preview, Android 4.1.2

### ISSUE THREAD

**Issue by rperezb** [owncloud/android#1319]
STEPS:
1.- Set the instant upload behavior to: delete
2.- Take a photo
3.- The photo is uploaded successfully
4.- Access to the folder where the photo is stored on the ownCloud app

CURRENT BEHAVIOR:
The photo is shown as downloaded, however, if you preview it there is an error: "The image cannot be shown"

LOGS
````
11-30 09:36:37.600  11714-14582/com.owncloud.android E/PreviewImageFragment﹕ File could not be loaded as a bitmap: /storage/sdcard0/owncloud/admin@docker.oc.solidgear.es%3Axxxx/InstantUpload/20151130_093530.jpg
````
EXPECTED BEHAVIOR 
The image is not shown as downloaded

(the error in the preview is fixed if you remove locally the file and download it again)

DEVICE:

Galaxy S3, 4.1.2
However, on an HTC one 4.3 works ok

**Comment by tobiasKaminsky** [comment]
Not for me (Nexus 4 [real device], Nexus 5 [emulator]).
When entering the folder there was a yellow circle badget and the file was just as not downloaded.

So, I am currently clueless...

**Comment by tobiasKaminsky** [comment]
Does a folder refresh helps?

**Comment by rperezb** [comment]
Yes, the yellow circle is shown, however, I think it shouldn't because the selected option is delete.
However, lets leave this to @masensio since here we reproduce it.

**Comment by masensio** [comment]
Hi @tobiasKaminsky,

I've checked this bug with a Nexus 9 Android 6, and I've reproduced it. 

The image is shown properly but the yellow circle is shown but there isn't any sync process running on this folder.  
On the other hand, if you refresh the folder the yellow circle continues there.
I'm reviewing this problem, looking for a solution.

**Comment by masensio** [comment]
Hi,
related with the yellow sync circle we have already fixed a similar bug in the release branch. 
I applied the same fix here with a cherry pick and it works fine. 

On the other hand, the preview problem in S3 device (4.1.2) is still present because is marked as downloaded, but I don't know why. I'm researching about it.

**Comment by masensio** [comment]
Hi again, 

this bug is fixed on the PR #1168: commit https://github.com/owncloud/android/commit/a7304351b1d8a821c64fcd0218106a0bcc5f723f

So I'll close it.

**Comment by tobiasKaminsky** [comment]
:+1: Thank you for bugfixing @masensio!
